deployment/tfhub/object-detection/base64/object-detection-base64-deploy.ipynb

```diff
@@ -0,0 +1,390 @@
+    "        \n",
+    "        return img\n",
+    "\n",
+    "    def filter_results(self, file, response, entity='Car', min_score=.2):\n",
+    "        unused_keys = ['detection_class_labels', 'detection_class_names']\n",
+    "        response = {key: value.numpy().tolist() for key, value in response.items()}\n",
+    "        response = {key: val for key, val in response.items() if key not in unused_keys}\n",
+    "        response['detection_class_entities'] = [v.decode() for v in response['detection_class_entities']]\n",
+    "\n",
+    "        entities = response['detection_class_entities']\n",
+    "        scores = response['detection_scores']\n",
+    "        bboxes = response['detection_boxes']\n",
+    "        result = {}\n",
+    "\n",
+    "        for i in range(len(entities)):\n",
+    "            if entities[i] == entity and scores[i] >= min_score:\n",
+    "                ymin, xmin, ymax, xmax = bboxes[i]\n",
+    "                result = {\n",
+    "                    'file': file,\n",
+    "                    'has_car': True,\n",
+    "                    'score': scores[i],\n",
+    "                    'bboxes': {'ymin': ymin, 'xmin': xmin, 'ymax': ymax, 'xmax': xmax}\n",
+    "                }\n",
+    "                break\n",
+    "\n",
+    "        if len(result) == 0:\n",
+    "            result = {\n",
+    "                'file': file,\n",
+    "                'has_car': False,\n",
+    "                'score': 0,\n",
+    "                'bboxes': {'ymin': 0, 'xmin': 0, 'ymax': 0, 'xmax': 0}\n",
+    "            }\n",
+    "        \n",
+    "        print(f\"Found {result['has_car']} object(s).\")\n",
+    "        return result\n",
+    "\n",
+    "    def run_detector(self, file, image_path):\n",
+    "        img = self.load_img(image_path)\n",
+    "        img = tf.image.convert_image_dtype(img, tf.float32)[tf.newaxis, ...]\n",
+    "        \n",
+    "        response = self.detector(img)\n",
+    "        result = self.filter_results(file, response)\n",
+    "        \n",
+    "        return result\n",
+    "\n",
+    "    def detect_objects(self, file, img_arr):\n",
+    "        start_time = time.time()\n",
+    "        image_path = self.handle_img(img_arr)\n",
+    "        result = self.run_detector(file, image_path)\n",
+    "        end_time = time.time()\n",
+    "\n",
+    "        print(f\"Inference time: {end_time - start_time}.\")\n",
+    "        return result\n",
+    "\n",
+    "    @verify_io\n",
+    "    def predict(self, data):\n",
+    "        file, img_arr = data\n",
+    "        result = self.detect_objects(file, img_arr)\n",
+    "\n",
+    "        return result\n",
+    "\n",
+    "    def describe(self):\n",
+    "        \"\"\"Return a description of the service.\"\"\"        \n",
+    "        return {\n",
+    "            \"method\": \"predict\",\n",
+    "            \"args\": f\"{self.img_base64}\",\n",
```

Comment:
We might want to remove `"args"` for this notebook, because the web UI model playground loads an entire base64-encoded image, which doesn't look great:

![Screen Shot 2022-12-01 at 12 37 18 PM](https://user-images.githubusercontent.com/96442646/205154766-b43800fd-92b8-416e-99b0-8267afdd99f5.png)

cc @MeetaDash

Reply:
Actually we might need to skip model playground entirely for this. I tried to run a prediction through the UI, and the webpage completely froze, possibly from loading the example.

Reply:
Sure. I added this after Manasi's request, and also noted that for base64 version the playground takes time to load - but I was able to run even so.
Nevertheless, I'll remove `describe()` and `example()` functions from base64 version :+1: